common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -1,34 +1,39 @@
-import { leafCommand, makeCommandInfo } from "../../framework/command";
 import {
   Extractor,
   ExtractorConfig,
+  ExtractorLogLevel,
   ExtractorResult,
   IConfigApiReport,
   IConfigDocModel,
   IConfigDtsRollup,
   IConfigFile,
 } from "@microsoft/api-extractor";
+import { createReadStream, createWriteStream } from "fs";
+import { leafCommand, makeCommandInfo } from "../../framework/command";
 
+import archiver from "archiver";
 import { createPrinter } from "../../util/printer";
-import { resolveProject } from "../../util/resolveProject";
 import path from "path";
 import { readFile } from "fs-extra";
 import { readdir } from "fs/promises";
-import { createReadStream, createWriteStream } from "fs";
-import archiver from "archiver";
+import { resolveProject } from "../../util/resolveProject";
 
 export const commandInfo = makeCommandInfo(
   "extract-api",
   "Runs api-extractor multiple times for all exports.",
-  {},
+  {
+    "subpath-doc-model": {
```

Comment:
default to true (or remove) once https://github.com/Azure/azure-sdk-for-js/blob/c8a0c166c87dee9f96e05cdad4ff1acba4f12d31/eng/scripts/stage-api-review-file.ps1#L26 supports multiple api.json files